specs/experimental/contracts/universal/safe-send.md

```diff
@@ -0,0 +1,151 @@
+# SafeSend
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
+
+- [Overview](#overview)
+- [Definitions](#definitions)
+  - [Forced ETH Transfer](#forced-eth-transfer)
+  - [Code Execution](#code-execution)
+- [Assumptions](#assumptions)
+  - [aSS-001: EVM selfdestruct behavior remains consistent](#ass-001-evm-selfdestruct-behavior-remains-consistent)
+    - [Mitigations](#mitigations)
+- [Dependencies](#dependencies)
+- [Invariants](#invariants)
+  - [iSS-001: ETH must be transferred without triggering recipient code](#iss-001-eth-must-be-transferred-without-triggering-recipient-code)
+    - [Impact](#impact)
+  - [iSS-002: Contract must self-destruct immediately upon construction](#iss-002-contract-must-self-destruct-immediately-upon-construction)
+    - [Impact](#impact-1)
+  - [iSS-003: Transfer must succeed for any recipient address](#iss-003-transfer-must-succeed-for-any-recipient-address)
+    - [Impact](#impact-2)
+  - [iSS-004: No state persists after construction](#iss-004-no-state-persists-after-construction)
+    - [Impact](#impact-3)
+- [Function Specification](#function-specification)
+  - [constructor](#constructor)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Overview
+
+SafeSend is a utility contract that enables forced ETH transfers to any address without triggering code
+execution at the recipient. It accomplishes this by using the `selfdestruct` opcode during construction,
+which transfers the contract's balance to the recipient without invoking any fallback or receive functions.
+
+## Definitions
+
+### Forced ETH Transfer
+
+A transfer of ETH that bypasses the recipient's code execution. Unlike standard ETH transfers via `call` or
+`transfer`, a [Forced ETH Transfer] does not invoke the recipient's `receive()` or `fallback()` functions,
+and cannot be reverted by the recipient.
+
+### Code Execution
```

Comment:
devinai Code execution is a useless definition - please don't define things that are obvious